### ISSUE TITLE
Cargo no longer allows wildcard depdency versions

### ISSUE THREAD

**Issue by Susurrus** [tailhook/rust-argparse#31]
The Readme should be updated to reflect this (as of Rust 1.6).

**Comment by tailhook** [comment]
Yeah. Actually I never found that message too useful (may be just for users new in rust). It was contributed by @jamperry. Should we just remove it now? Assuming everybody knows cargo now, and the text with right version is printed on a crates.io page.

**Comment by Susurrus** [comment]
I would just defer to that page then for usage instructions then.

**Comment by pradyunsg** [comment]
I suggest putting a cargo version badge as well. It would make sense to link to the cargo page from that badge.
